olad/www/Makefile.mk

```diff
@@ -76,10 +77,10 @@ dist_angular_DATA = \
 dist_bootjs_DATA = \
     olad/www/new/libs/bootstrap/js/bootstrap.min.js
 dist_bootfonts_DATA = \
-    olad/www/new/libs/bootstrap/fonts/glyphicons-halflings-regular.woff \
+    olad/www/new/libs/bootstrap/fonts/glyphicons-halflings-regular.eot \
     olad/www/new/libs/bootstrap/fonts/glyphicons-halflings-regular.svg \
     olad/www/new/libs/bootstrap/fonts/glyphicons-halflings-regular.ttf \
-    olad/www/new/libs/bootstrap/fonts/glyphicons-halflings-regular.eot \
+    olad/www/new/libs/bootstrap/fonts/glyphicons-halflings-regular.woff \
```

Comment:
I thought this seemed familiar; as we've both worked out I'd done it here first https://github.com/OpenLightingProject/ola/pull/809 .